xtables-10.h

```diff
@@ -70,14 +70,14 @@ fw3_xt_merge_match_options(struct xtables_globals *g, struct xtables_match *m)
 {
 	if (m->x6_options)
 		g->opts = xtables_options_xfrm(g->orig_opts, g->opts,
-									   m->x6_options, &m->option_offset);
-
-	if (m->extra_opts)
+						m->x6_options, &m->option_offset);
+	else if (m->extra_opts)
```

Comment:
Can you better explain this? With this change it might happen that extra_opts is never called... Is it intended?

Reply:
The variable m->extra_opts can have a non-NULL value only if m->x6_options has an NULL value. In case of an NULL value of m->extra_opts the call is not required.